docker-compose.yml

```diff
@@ -55,8 +55,6 @@ services:
     command: rails s -p 3000 -b 0.0.0.0
```

Comment:
Maybe these are wanted for development, to be able to run the tests with your local code?

It may be ok to remove them here, but when we get into this repo next and are making changes, we may want a mode available that maps our local code into the images to run the tests

Reply:
I totally see your point but the change serves multiple subtle purposes:
- It's a limitation of snipping out the dev/test gems. Without removal of the mounted dir, the gemfile with both test and dev gems is mounted and bundler blows up since it can't find the gems it needs. 
- Without relying strictly on the built image, we don't really know what is being tested since your local env could be different from the image
- It's a tiny speedup since it just needs to launch the pre-built image

Reply:
makes sense, but maybe we need a follow-up issue to provide a way for devs to run these tests with their local code in a realistic way. it's fine if this is big enough to be out of scope for this effort. it might be worth adding a note to the contributing that explains that when you run these tests, you're not using your local code unless you've built the image first.

Reply:
@sgnn7 can you please update the CONTRIBUTING with instructions to build before running the tests locally after making changes? with that, I think this PR is gtg